validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/BlobsetSequences.java

```diff
@@ -198,7 +198,7 @@ public void endpoint_connection_success_alternate() {
 
   @Test
   @Description("Restart and connect to same endpoint and expect it returns.")
-  @FeatureStage(PREVIEW)
+  @Feature(stage = PREVIEW)
```

Comment:
DO we need category?

Reply:
Right now, if no category is defined then it just defaults to "unknown."  Basically it comes down to the behavior we want for a test that has no particular category defined.  Either we say 1) it has to have a category, 2) for "no category" then we leave out the @Feature annotation, or 3) defaults to something.  Doesn't really matter to me what behavior it has - it's all relatively easy to change.  The only enforcement there is right now is that the end result has to have A category in it, but it's undetermined exactly how/when that is specified in all cases.

Reply:
I meant should we fill in category values that Nour is generating. But we can revisit this when we settle the enums, too.